README.md

```diff
@@ -5,18 +5,24 @@ If you know M-BoF, you don't need to know anything else.
 
 0. Install Vagrant - https://www.vagrantup.com/
 0. Start Vagrant
-   - `cd hacks_mbof`
+   - `cd ocellus`
    - `vagrant up`
    - `vagrant ssh`
 0. Install Bower packages
    - `cd /vagrant/mbofui`
    - `bower install`
 
 # Development
-0. Initialize and start app server
-    - `cd /vagrant`
+0. Prepare the application configuration
+    - `cd /vagrant/hacks_mbof`
+    - `cp settings.py.template settings.py`
+    - Edit `settings.py` to set various values:
+        - `SECRET_KEY` A secret key can be generated at http://www.miniwebtool.com/django-secret-key-generator/
+0. Initialize Database
+    - 'cd /vagrant'
     - `python manage.py migrate`
     - `python manage.py loaddata mbof/fixtures/dev_data.json`
+0. Start the application server
     - Run the development app server
         - _**Without**_ remote user specified
             - `python manage.py runserver`
```

Comment:
Everything seemed OK this time, but when I tried to go to http://localhost:18000/, I'd be redirected to https://localhost:18000/, which would fail with errors in the log, like:

```
[31/May/2016 17:27:58] code 400, message Bad HTTP/0.9 request type ('\x16\x03\x01\x00\xc0\x01\x00\x00\xbc\x03\x03\xbd\xb9e\xb3x\x1e\x91\xe0,O\xa8d\xfa\xf2?\x1cYPj\xf4\xec\xcd\xa7\xdc\x1d}\xd0')
You're accessing the development server over HTTPS, but it only supports HTTP.
[31/May/2016 17:27:58] code 400, message Bad request syntax ('\x16\x03\x01\x00\xc0\x01\x00\x00\xbc\x03\x03\x9e\x9c\xee\x1cd(bc\xf7B\xd9\x1c-\x81\x1c\x1fQ7\xec\xeca\x0f\x1c\x14\xadD+>\xb9a\xf3\xc7\x00\x00\x1c\xc0+\xc0/\xcc\xa9\xcc\xa8\xcc\x14\xcc\x13\xc0')
You're accessing the development server over HTTPS, but it only supports HTTP.
[31/May/2016 17:27:58] code 400, message Bad request syntax ('\x16\x03\x01\x00\x9b\x01\x00\x00\x97\x03\x02\xd1h\xe3\xcee\r\xbes\xc9-\xa7\t\xb5 %\x15<\xc4\xcd\xf88\xa1q\xfb\xad\xe5p\x17\xae\xdc\xc1\x0f\x00\x00\x10\xc0')
You're accessing the development server over HTTPS, but it only supports HTTP.
```